.github/workflows/dev_pr.yml

```diff
@@ -28,9 +28,8 @@ on:
       - edited
       - synchronize
 
-concurrency:
-  group: ${{ github.repository }}-${{ github.ref }}-${{ github.workflow }}
-  cancel-in-progress: true
+# NOTE: not using the "cancel-in-progress" feature here as the group key
+# does not have enough information for linking it to a particular PR
```

Comment:
Do we have group collisions? Could we inspect the different github variables and come up with a better key?

Reply:
See comment: the current key is adequate for a "push_request" event but not for a "push_request_target" event.
Example:
https://github.com/apache/arrow/actions/runs/914966579

Given that "Dev PR" should be a fast workflow, I'm not sure it's worth trying to come up with an alternative.

Reply:
Thanks for the clarification!